cumulusci/tasks/robotframework/robotframework.py

```diff
@@ -97,6 +108,23 @@ def _run_task(self):
             os.path.join(self.working_path, options.get("outputdir", ".")), os.getcwd()
         )
 
+        # get_namespace will potentially download sources that have
+        # yet to be downloaded. For these downloaded sources we'll add
+        # the cached directories to PYTHONPATH before running.
+        source_paths = {}
+        for source in self.options["sources"]:
+            try:
+                source_config = self.project_config.get_namespace(source)
+                source_paths[source] = source_config.repo_root
+            except NamespaceNotFoundError:
+                raise TaskOptionsError(f"robot source '{source}' could not be found")
+
+        # replace namespace prefixes with path to cached folder
+        for i, path in enumerate(self.options["suites"]):
+            prefix, _, path = path.rpartition(":")
+            if prefix in source_paths:
+                self.options["suites"][i] = f"{source_paths[prefix]}/{path}"
```

Comment:
Does this work in Windows, or do we need to join with os.sep?

Side note, I've never heard of rpartition before!

Reply:
Forward slashes work on windows, but I'm not sure what the behavior is if part of the path includes forward slashes and part has backward slashes (eg: if they used forward slashes in cumulusci.yml to define sources). 

I'll use `os.path.join`. Good catch.